sources/models/shared-defaults/oci-defaults-docker-cri-resource-limits.toml

```diff
@@ -0,0 +1,3 @@
+[settings.oci-defaults.resource-limits.max-open-files]
+hard-limit = 4086
+soft-limit = 1024
```

Comment:
These are what the current settings are right? Do we want to bring them in line with the defaults for our other variants?

Reply:
Yeah these are as per the current settings for ecs variants. Ok I will update them same as k8s variants.